content/Foundation/Summit/2023/submit.pml

```diff
@@ -0,0 +1,32 @@
+<!--<title>submit</title>-->
```

Comment:
At first I didn't understand this page as it looks empty on the preview. ButI checkout the branch and checked there, it looks very good! I didn't try to submit a fake talk as I would have to come up with a fake pdf to submit :) But it looks very good overall.